test/hotspot/jtreg/compiler/compilercontrol/share/scenario/Executor.java

```diff
@@ -57,16 +56,13 @@ public class Executor {
     /**
      * Constructor
      *
-     * @param isValid      shows that the input given to the VM is valid and
-     *                     VM shouldn't fail
      * @param vmOptions    a list of VM input options
      * @param states       a state map, or null for the non-checking execution
      * @param jcmdCommands a list of diagnostic commands to be preformed
      *                     on test VM
      */
-    public Executor(boolean isValid, List<String> vmOptions,
-            Map<Executable, State> states, List<String> jcmdCommands) {
-        this.isValid = isValid;
+    public Executor(List<String> vmOptions, Map<Executable, State> states,
+                    List<String> jcmdCommands) {
```

Comment:
`compiler.compilercontrol.jcmd.StressAddJcmdBase` extends `Executor`, so its constructor needs to be updated.

Reply:
I see you just addressed this in [f477f9e](https://github.com/openjdk/jdk/pull/14538/commits/f477f9e2d5c38d248fa22ff0550b84390fa075ef), thanks!

Reply:
Thanks, I noticed as well and updated the file in parallel to you review.